php_phongo.h

```diff
@@ -182,26 +182,26 @@ bool php_phongo_manager_unregister(php_phongo_manager_t* manager);
 	} while (0)
 #endif
 
-#define PHONGO_GET_PROPERTY_HASH_INIT_PROPS(is_debug, intern, props, size) \
-	do {                                                                   \
-		if (is_debug) {                                                    \
-			ALLOC_HASHTABLE(props);                                        \
-			zend_hash_init((props), (size), NULL, ZVAL_PTR_DTOR, 0);       \
-		} else if ((intern)->properties) {                                 \
-			(props) = (intern)->properties;                                \
-		} else {                                                           \
-			ALLOC_HASHTABLE(props);                                        \
-			zend_hash_init((props), (size), NULL, ZVAL_PTR_DTOR, 0);       \
-			(intern)->properties = (props);                                \
-		}                                                                  \
+#define PHONGO_GET_PROPERTY_HASH_INIT_PROPS(is_temp, intern, props, size) \
```

Comment:
Since `__serialize` will also return a newly allocated HashTable, this is no longer just used by the `get_debug_info` handlers. `is_temp` seemed like a more accurate name.